code-challenges.test.js

```diff
@@ -15,35 +15,143 @@
+        expect(minMax(nums2)).toEqual([5, 109])
+    })
+})
+            // FAIL  ./code-challenges.test.js
+            // minMax
+            //     ✕ takes an array of numbers and returns an array of the minimum and maximum numbers in that order (1 ms)
+            //   ● minMax › takes an array of numbers and returns an array of the minimum and maximum numbers in that order
+            //     ReferenceError: minMax is not defined
 
 
 // b) Create the function that makes the test pass.
 
+//Create a function called minMax that takes in an array of numbers as an argument in the parameter
+ //create a variable called newArr to contain the values of the new array
+ //push the smalles number of the array to newArr using Math.min and a spread operator
+ //push the largest number of the array to newArr using Math.max and a spread operator
+ //return the newArr
+
+
+ const minMax = (array) => {
+    let newArr = []
+    newArr.push(Math.min(...array))
+    newArr.push(Math.max(...array))
+    return newArr
+  }
+
+
+            //   PASS  ./code-challenges.test.js
+            //   minMax
+            //   ✓ takes an array of numbers and returns an array of the minimum and maximum numbers in that order (1 ms)
+
+
+
+
 
 // --------------------3) Create a function that takes in two arrays as arguments and returns one array with no duplicate values. STRETCH: Use the spread operator to pass in a dynamic number of arguments.
 
 // a) Create a test with an expect statement using the variables provided.
 
-const testArray1 = [3, 7, 10, 5, 4, 3, 3]
-const testArray2 = [7, 8, 2, 3, 1, 5, 4]
-// Expected output: [3, 7, 10, 5, 4, 8, 2, 1]
+describe("noDuplicate", () => {
+    it("takes in two arrays as arguments and returns one array with no duplicate values", () => {
+
+        const testArray1 = [3, 7, 10, 5, 4, 3, 3]
+        const testArray2 = [7, 8, 2, 3, 1, 5, 4]
+        // Expected output: [3, 7, 10, 5, 4, 8, 2, 1]
+        expect(noDuplicate(testArray1, testArray2)).toEqual([3, 7, 10, 5, 4, 8, 2, 1])
+
+    })
+})
+            // FAIL  ./code-challenges.test.js
+            // noDuplicate
+            //     ✕ takes in two arrays as arguments and returns one array with no duplicate values (1 ms)
+            //   ● noDuplicate › takes in two arrays as arguments and returns one array with no duplicate values
+            //     ReferenceError: noDuplicate is not defined
+
+
+
+
+
 
 
 // b) Create the function that makes the test pass.
+
+//Create a function called noDuplicate that takes in two arrays of numbers
+    //create a variable newArr that combines the two arrays using spread operators
+    //create a new variable called removeDuplicates that contains a Set of the combined array removing the duplicate values
+    //return removeDuplicates array
+
+const noDuplicate = (array1, array2) => {
+    let newArr = [...array1, ...array2]
+    let removeDuplicates = [...new Set(newArr)]
+    return removeDuplicates
+}
```

Comment:
Very cool and very close. We are able to use the rest /spread operator in the parameter declaration and then call on all possible arguments in a line very similar to your line 151.